src/benchmark.cc

```diff
@@ -520,6 +524,15 @@ size_t RunSpecifiedBenchmarks(BenchmarkReporter* display_reporter,
   return benchmarks.size();
 }
 
+namespace {
+// stores the time unit benchmarks use by default
+TimeUnit default_time_unit = kNanosecond;
+}  // namespace
+
+TimeUnit GetDefaultTimeUnit() { return default_time_unit; }
+
+void SetDefaultTimeUnit(TimeUnit unit) { default_time_unit = unit; }
```

Comment:
it's a shame we  can't catch this being called at the wrong time.  in `State` we can check `!started_ && !finished_` but i don't think we expose that anywhere, and you don't have `State` here anyway.

i don't think there's anything to be done, but i wanted to flag it in case you had an idea.

Reply:
Can't think of a good solution either. Maybe putting it in the `internal` namespace thus not recommending the usage of this (or just removing it from the header). But I'd rather leave the option to set this programatically.